src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/transport/ParsedMessage.kt

```diff
@@ -87,6 +87,7 @@ class ParsedMessage private constructor(
 
     interface Builder<out T : Builder<T>> : Message.Builder<T> {
         val type: String
+        val isTypeSet: Boolean
```

Comment:
Should we add similar properties for Parsed builder implementation

Reply:
All parsed builders extend this interface. So, they have it already